p2p/proto/block.proto

```diff
@@ -72,10 +78,12 @@ message BlockBodiesResponse {
     optional BlockID id = 1; // may not appear if Fin is sent to end the whole response
 
     oneof body_message {
-        StateDiff  diff    = 2;
-        Classes    classes = 3;
-        BlockProof proof   = 4;
-        Fin        fin     = 5;
+        StateDiff    diff         = 2;
+        Classes      classes      = 3;
+        BlockProof   proof        = 4;
+        Transactions transactions = 5;
+        Receipts     receipts     = 6;
```

Comment:
Why should transactions and receipts be included in the response stream?
Why not just use separate queries for them (as in the current spec version)?

Reply:
I added the transactions and receipts as part of the block body for completeness as it was represented by the block returned by the feeder gateway. However, you are correct we can fetch them separately. 

Based on the same reasoning I don't think we should have a body message at all as we have state commitment which can be used to verify `StateDiff` and `Classes`, while the `BlockProof` can be included in the `Header`. Also `StateDiff`, `Classes` and `Block` proof are already sent separately just wrapped in a body message.

This also ties in with having multiple protocols instead of one. I may be missing some context but I cannot see under which situation a node would be interested in just transactions/events/receipts/headers etc. How would the node verify the validity of the single message which they retrieved without also retrieving other information such as block header, state diff transactions etc? And why would RPC not be enough (cc:@joshklop)? We can decide to add web sockets for whatever data the user is interested in. In my opinion, we should have a single protocol.

Please see this convo for more [context](https://github.com/starknet-io/starknet-p2p-specs/pull/9/files/3da1088ba741dce3d52c3a67b7ffe163674437c1#r1390334671).